frontend/app/routes/$lang+/_protected+/home.tsx

```diff
@@ -22,7 +23,7 @@ import { getUserOrigin } from '~/utils/user-origin-utils.server';
 
 export const handle = {
   i18nNamespaces: getTypedI18nNamespaces('index', 'gcweb'),
-  pageIdentifier: 'CDCP-0001',
+  pageIdentifier: pageIds.protected.personalInformation.home,
```

Comment:
Since this page is outside the "Personal Information" section (corresponding change would need to be made in `page-ids.json`),

```suggestion
  pageIdentifier: pageIds.protected.home,
```

Reply:
Makes sense.

Implemented in commit fbfdd8b4c5d63d7d23371845a0de603f25b57d23.